tests/apps/stateapp/components/redis_messagebus.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: dapr.io/v1alpha1
```

Comment:
JFYI - in e2e test, [this config](https://github.com/dapr/dapr/blob/master/tests/config/dapr_redis_state.yaml) is used.